kubernetes/samples/scripts/common/domain-template.yaml

```diff
@@ -75,8 +93,4 @@ spec:
     replicas: %INITIAL_MANAGED_SERVER_REPLICAS%
   # The number of managed servers to start for unlisted clusters
   # replicas: 1
-  # 
-  experimental:
```

Comment:
How about commenting these out instead of removing, so that it can be experimented easier?